js/src/App.jsx

```diff
@@ -1,30 +1,70 @@
 import { ThemeProvider } from '@mui/material/styles';
 import CssBaseline from '@mui/material/CssBaseline';
-import { Routes, Route } from 'react-router-dom';
+import { BrowserRouter as Router, Routes, Route, Navigate } from 'react-router-dom';
 import theme from './theme';
 
 import Home from './pages/Home';
 import Products from './pages/Products';
-import LoginPage from './pages/LoginPage';
 import ProtectedRoute from './config/ProtectedRoute';
 import ForgotPassword from './pages/ForgotPassword';
 import AuthContainer from './components/auth/AuthContainer';
+import AdminDashboard from './pages/AdminDashboard';
+import DriverDashboard from './pages/DriverDashboard';
+import { useAuthStore } from './store/useStore';
 
 const App = () => {
-  const protectedRoutes = [
-    { path: '/', element: <Home /> },
-    { path: '/products', element: <Products /> },
-  ];
+  const { user } = useAuthStore();
 
   return (
     <ThemeProvider theme={theme}>
       <CssBaseline />
       <Routes>
+        {/* Public Routes */}
         <Route path="/login" element={<AuthContainer />} />
```

Comment:
dont hardcode paths, all of them should be in constant file and import here

Reply:
made into a const file.